_non-cosmos/ethereum/assetlist.json

```diff
@@ -1222,6 +1222,16 @@
       "name": "Gelotto BEAST",
       "display": "beast",
       "symbol": "BEAST",
+      "traces": [
+        {
+          "type": "additional-mintage",
```

Comment:
this trace would be invalid. I see no need to update the ethereum assetlist here